.github/workflows/ci.yml

```diff
@@ -72,6 +72,13 @@ jobs:
           SKIP=pip-compile pre-commit run --all-files --show-diff-on-failure
         working-directory: ./some-package
 
+      - name: Upload generated project as CI Artifact
+        uses: actions/upload-artifact@v3
+        with:
+          name: cookiecutter-pypackage-example
+          path: |
+            ./some-package
```

Comment:
artifact is 94.6 MB in github ui and 45,9 MB when downloaded (= zip'ed). most of it (97%, 91 MB) is because of the project-venv, and 55% of this is used by the lib packages `virtualenv`, `setuptools` & `pip`

![grafik](https://user-images.githubusercontent.com/32995541/182375036-4458b666-9b20-43ad-b6fa-68a1a7eb5842.png)
![grafik](https://user-images.githubusercontent.com/32995541/182375095-bbb513e0-3684-4aa4-82ec-7859ef7e2937.png)


because the ci already uses pinned versions for installing all dependencies into said venv and the ci run already has python version logged by python action, i'd prefer to ditch the inclusion of it.

Reply:
309KB zipped now! thanks for the info, yes youre rights the versions are already pinned and conclusively reproducible